src/OrchardCore.Modules/OrchardCore.AdminMenu/Services/IAdminMenuPermissionService.cs

```diff
@@ -0,0 +1,11 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using OrchardCore.Security.Permissions;
+
+namespace OrchardCore.AdminMenu.Services
+{
+    public interface IAdminMenuPermissionService
+    {
+        Task<IEnumerable<Permission>> GetPermissionsAsync();
+    }
+}
```

Comment:
Move this to OrchardCore.AdminMenu.Abstractions. At the same time, I'm not sure if it should not be a more globally acessible interface.